packages/components/src/components/post-mainnavigation/post-mainnavigation.scss

```
@@ -11,18 +11,20 @@ $nav-height: var(--post-core-dimension-56);
 post-mainnavigation {
   // reset links and buttons
   post-list-item {
-    a {
+    > a {
       text-decoration: none;
       border-radius: 0;
     }
 
-    button {
+    > button,
+    post-megadropdown-trigger button {
```

I think this is not necessary, there will only be buttons in the post-megadropdown-trigger. 
```suggestion
    post-megadropdown-trigger button {
```